http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpCromwellDAOSpec.scala

```diff
@@ -42,7 +42,7 @@ class HttpCromwellDAOSpec extends AnyFlatSpec with Matchers with LeonardoTestSui
       """.stripMargin
 
     val okCrom = Client.fromHttpApp[IO](
-      HttpApp(_ => IO.fromEither(parse(response)).flatMap(r => IO(Response(status = Status.Ok).withEntity(r))))
+      HttpApp(_ => IO.fromEither(parse(response)).flatMap(r => IO(Response(status = Status.BadGateway).withEntity(r))))
     )
```

Comment:
Why was this changed from `Ok` to `BadGateway` here?

Reply:
Ah, see the TODO in `HttpCromwellDAO`. This test is trying to simulate an unhealthy cromwell. But now we treat any OK status as healthy. So updating the test to return 502 (which is actually the status code we get when the cromwell pod is down).